kwanghwi/myapp/db/migrations/20221220062656_create_likes_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE likes (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  user_id INT NOT NULL,
+  post_id INT NOT NULL,
+  CONSTRAINT likes_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
+  CONSTRAINT likes_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id)
+);
```

Comment:
한명의 유저는 동일 게시글에 2번의 좋아요를 할 수 없습니다.
user_id와 post_id를 묶어 unique 제약 조건을 걸어줄 수 있습니다.
```suggestion
CREATE TABLE likes (
  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  user_id INT NOT NULL,
  post_id INT NOT NULL,
  CONSTRAINT likes_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
  CONSTRAINT likes_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id),
  CONSTRAINT likes_ukey UNIQUE (user_id, post_id)
);
```